gr-channels/lib/sincostable.h

```diff
@@ -19,17 +19,17 @@ class sincostable
             d_cos[i] = ::cos(2 * GR_M_PI * i / tbl_size);
         }
     }
-    const float sin(float x)
+    float sin(float x)
```

Comment:
the intent of the author was to write `float sin(float x) const`, so this change is not an improvement. Fix by moving const to end (the whole existence of this class is more than questionable, but that's another separate issue)

Reply:
Done. (Note: changes are committed but not yet pushed, partly because I see I missed one. But since I can't batch reply, I'm going to post these as a way for me to check off that I've made the changes. New push should be up in a half hour or less.)